catalog/app/containers/Auth/SSOOkta.js

```diff
@@ -31,7 +32,7 @@ export default ({ mutex, next }) => {
   const { push: notify } = Notifications.use()
   const { urls } = NamedRoutes.use()
 
-  const handleClick = () => {
+  const handleClick = React.useCallback(() => {
```

Comment:
Just wondering, when is `React.useCallback` needed? When you pass the function as an attribute to a React element?

Does `handleClick` in `SSOGoogle.js` need it?

Reply:
it's needed when a function is used as a prop, correct (to minimize rerenders) or as a dependency for another useCallback

> Does handleClick in SSOGoogle.js need it?

it's a factory returning a callback, so we cant do anything with it in this simple way (some memoization would help, but i don't think it's worth it there)